src/index.html

```diff
@@ -17,11 +17,454 @@
+              <div class="star"></div>
+              <div class="star"></div>
+              <div class="star"></div>
+              <div class="star"></div>
+              <div class="star"></div>
+            </div>
+            <p class="card__reviews-count">Reviews: 5</p>
+          </div>
+
+          <p class="card__details-price">
+            Price:
+            <span>$2,199</span>
+          </p>
+        </div>
+
+        <a
+          href="#"
+          class="card__button-buy"
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </div>
+
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__product">
+          <img
+            src="./images/imac.jpeg"
+            alt="Imac Image"
+            class="card__product-image"
+          />
+        </div>
+
+        <div class="card__details">
+          <h1 class="card__details-name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h1>
+
+          <p class="card__details-code">Product code: 195434</p>
+
+          <div class="card__reviews">
+            <div class="card__reviews-stars card__reviews-stars--4">
+              <div class="star"></div>
+              <div class="star"></div>
+              <div class="star"></div>
+              <div class="star"></div>
+              <div class="star"></div>
+            </div>
+            <p class="card__reviews-count">Reviews: 5</p>
+          </div>
+
+          <p class="card__details-price">
+            Price:
+            <span>$2,199</span>
+          </p>
+        </div>
+
+        <a
+          href="#"
+          class="card__button-buy"
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </div>
+
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__product">
+          <img
+            src="./images/imac.jpeg"
+            alt="Imac Image"
+            class="card__product-image"
+          />
+        </div>
+
+        <div class="card__details">
+          <h1 class="card__details-name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h1>
+
+          <p class="card__details-code">Product code: 195434</p>
+
+          <div class="card__reviews">
+            <div class="card__reviews-stars card__reviews-stars--4">
+              <div class="star"></div>
+              <div class="star"></div>
+              <div class="star"></div>
+              <div class="star"></div>
+              <div class="star"></div>
+            </div>
+            <p class="card__reviews-count">Reviews: 5</p>
+          </div>
+
+          <p class="card__details-price">
+            Price:
+            <span>$2,199</span>
+          </p>
+        </div>
+
+        <a
+          href="#"
+          class="card__button-buy"
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </div>
```

Comment:
The product cards are repeated with the same content. If this is not intentional, consider using a template engine or JavaScript to dynamically generate unique content for each card. This will help in maintaining the DRY (Don't Repeat Yourself) principle and make the code easier to manage.